Library/Homebrew/cmd/update.sh

```diff
@@ -378,19 +381,34 @@ EOS
   for DIR in "$HOMEBREW_REPOSITORY" "$HOMEBREW_LIBRARY"/Taps/*/*
   do
     [[ -d "$DIR/.git" ]] || continue
+    cd "$DIR" || continue
+
+    TAP_VAR="$(repo_var "$DIR")"
+    declare PREFETCH_REVISION"$TAP_VAR"="$(read_current_revision)"
```

Comment:
@ilovezfs I also assumed it would catch this but it doesn't verify the update actually happened until my change in https://github.com/Homebrew/brew/commit/b9d5b11f321a359b462aa2c353c2bdb483930fb3.